pom.xml

```diff
@@ -1257,7 +1258,7 @@
       <plugin>
         <groupId>org.codehaus.mojo</groupId>
         <artifactId>versions-maven-plugin</artifactId>
-        <version>2.1</version>
+        <version>2.3</version>
         <reportSets>
```

Comment:
Just curious: did you check the resulting artifacts ? It looks good to me anyway.